curp/src/server/curp_node.rs

```diff
@@ -289,7 +289,7 @@ impl<C: 'static + Command> CurpNode<C> {
             ))
         };
 
-        run_gc_tasks(Arc::clone(&cmd_board), Arc::clone(&spec_pool));
+        run_gc_tasks(Arc::clone(&cmd_board), Arc::clone(&spec_pool), &curp_cfg);
```

Comment:
We can pass the `gc_interval` to the `run_gc_tasks` directly. Hence, we don't need to wrap the `curp_cfg` by `Arc`.